src/lib/utils/drupalClient.ts

```diff
@@ -39,4 +37,5 @@ export const fetcher = async (input: RequestInfo, init?: RequestInit) => {
 
 export const drupalClient = new DrupalClient(baseUrl, {
   fetcher,
+  useDefaultResourceTypeEntry: true,
```

Comment:
setting this to true seems to increase build speed locally by `278548 ms`.

Without: `├ ● /[[...slug]] (1435969 ms)`
With: `├ ● /[[...slug]] (1157421 ms)`